lib/kitchen/driver/openstack.rb

```diff
@@ -101,6 +101,23 @@ def destroy(state)
         state.delete(:hostname)
       end
 
+      def env_cmd(cmd)
+        # if env is defined, add env terms when execute cmd.
+        is_sudo = cmd.start_with?('sudo -E')
+        naked_cmd = is_sudo ? cmd[8..-1] : cmd # remove sudo -E
+        env = 'env'
+        unless config[:env].nil?
+          config[:env].each do |env_k, env_v|
+            env << " #{env_k}=#{env_v}"
+          end
+        end
+        if env == 'env'
+          cmd
+        else
+          is_sudo ? "sudo -E #{env} #{naked_cmd}" : "#{env} #{cmd}"
```

Comment:
Does putting environment variables after the "sudo -E" behave differently? The [parent class' env_cmd](https://github.com/test-kitchen/test-kitchen/blob/master/lib/kitchen/driver/ssh_base.rb#L133) just prepends the environment variables so it doesn't have to do any parsing of sudo commands.